elements/gnome-sdk-sdk/cogl-endless-arm.bst

```diff
@@ -11,10 +11,10 @@
 kind: autotools
 
 sources:
-- kind: git_tag
-  url: gitlab_gnome_org:GNOME/cogl.git
-  track: cogl-1.22
-  ref: 1.22.6-22-g122b390bbf4bf4283ef2a94f25a44ed23ead3e05
+sources:
+- kind: tar
+  url: download_gnome_org_sources:cogl/1.22/cogl-1.22.6.tar.xz
+  ref: 6d134bd3e48c067507167c001200b275997fb9c68b08b48ff038211c8c251b75
```

Comment:
I'm not really familiar with this, but why we are not using 1.22.6 tag in cogl here?

Reply:
Sorry, just saw the description.

Reply:
> I don't see any changes for fcitx here. An I missing something?

The new ref for gnome-sdk.bst includes this commit: https://gitlab.gnome.org/GNOME/gnome-build-meta/-/commit/209e9ad8ca2463a478d9375a00fa27b82bc70b45. And now you mention it, that would be a useful thing to stick in the description :)

Reply:
That's to keep it in line with what's upstream at the commit referred to in gnome-sdk.bst. I'm not sure why it was using git_tag originally since upstream uses a tar file :)